### PR TITLE
SB-25552 - UCI Outbound Deployment

### PR DESCRIPTION
**Pull request by shiva-rakshith**
<table>
<td>
  <a href="https://project-sunbird.atlassian.net/browse/SB-25552" title="SB-25552" target="_blank">StorySB-25552</a>  UCI - Deployment
  </td></table>
  <br />
 

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

### Type of change

Please choose appropriate options.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Enhancement (additive changes to improve performance)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Please describe the tests that you ran to verify your changes in the below checkboxes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Ran Test A
- [ ] Ran Test B

**Test Configuration**:
* Software versions:
* Hardware versions:

### Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by shiva-rakshith** [comment]
@keshavprasadms Please review the PR.

File: kubernetes/helm_charts/core/outbound/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          configMap:
+            name: {{ .Chart.Name }}-xml-config    
+      containers:
+      - name: {{ .Chart.Name }}
+        image: "{{ .Values.dockerhub }}/{{ .Values.repository }}:{{ .Values.image_tag }}"
+        imagePullPolicy: Always
+        env:
+        - name: JAVA_OPTIONS
+          value: {{ .Values.env.javaoptions | quote }}
+        envFrom:
```
Comment (keshavprasadms) [inline kubernetes/helm_charts/core/outbound/templates/deployment.yaml:37]: Similar comment as before. Please check if both env and volumes are required for the configmap and update accordingly.

Comment (shiva-rakshith) [reply kubernetes/helm_charts/core/outbound/templates/deployment.yaml:34]: Made suggested changes.

**Comment by keshavprasadms** [comment]
@shiva-rakshith Please resolve conflicts

**Comment by shiva-rakshith** [comment]
> @shiva-rakshith Please resolve conflicts

I do not have the write access to the repository, not able to resolve conflicts.

**Comment by shiva-rakshith** [comment]
> @shiva-rakshith Please resolve conflicts

Resolved the conflicts.

File: ansible/roles/stack-sunbird/defaults/main.yml [hunk 2]
```diff
@@ -801,7 +804,7 @@ dial_liveness_readiness:
     periodSeconds: 15
     timeoutSeconds: 5
     failureThreshold: 2
-
+
```
Comment (keshavprasadms) [inline ansible/roles/stack-sunbird/defaults/main.yml:807]: I dont know if this is a tab or white spaces. Either way, please remove the extra spaces / tab.

Comment (shiva-rakshith) [reply ansible/roles/stack-sunbird/defaults/main.yml]: Removed Tab space.

**Review by keshavprasadms — APPROVED** [review]

**Comment by keshavprasadms** [comment]
Looks good!

**Pull request merged**